torchci/pages/api/drci/drci.ts

```diff
@@ -62,7 +62,8 @@ export async function updateDrciComments(octokit: Octokit, repo: string = "pytor
     );
 
     const workflowsByPR = reorganizeWorkflows(recentWorkflows);
-    await addMergeBaseCommits(octokit, workflowsByPR);
+    const head = await get_head_branch(repo);
```

Comment:
Not really an async function, but sure
```suggestion
    const head = get_head_branch(repo);
```